src/main/java/com/github/webdriverextensions/junitrunner/WebDriverRunner.java

```diff
@@ -519,6 +520,13 @@ public BrowserConfiguration(Annotation annotation) {
                 desiredCapabilities = addCapabilities(desiredCapabilities, desiredCapabilitiesJsonMap);
             }
 
+            if( browserName.equals(BrowserType.CHROME) && System.getProperty(WebDriverProperties.CHROME_BINARY_PROPERTY_NAME) != null) {
```

Comment:
Shouldn't the chrome binary driver only be applied when creating a local driver with `new ChromeDriver(desiredCapabilities)`?
Should it also be applied when creating a remote driver with `new RemoteWebDriver(url, desiredCapabilities);`?

If so this code block should be placed at https://github.com/dve/webdriverextensions/blob/a1592cc82b73c8ca76745c2bdd693715c95d8da9/src/main/java/com/github/webdriverextensions/junitrunner/WebDriverRunner.java#L549